### PR TITLE
Let other modules import chat, and chat commands.

### PR DESCRIPTION
**Pull request by peterbraden**
Hoprd will want to import chat commands for the browser UI, and also the chat functionality.

### CODE REVIEW

File: commands/commands.spec.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+  it('ping', async () => {
+    let mockNode: any = jest.fn()
+    mockNode.bootstrapServers = []
+    mockNode.ping = jest.fn()
+    let mockPeerId = '16Uiu2HAkyXRaL7fKu4qcjaKuo4WXizrpK63Ltd6kG2tH6oSV58AW'
+
+    let cmds = new mod.Commands(mockNode)
+    await cmds.ping.execute(mockPeerId)
+    expect(mockNode.ping).toHaveBeenCalled()
```
Comment (jjperezaguinaga) [inline commands/commands.spec.ts:26]: Really like this approach 👍

File: commands/index.ts [hunk 2]
```diff
@@ -16,7 +16,7 @@ import Version from './version'
 import Tickets from './tickets'
 import IncludeRecipient from './includeRecipient'
 
-export default class Commands {
+export class Commands {
```
Comment (jjperezaguinaga) [inline commands/index.ts:19]: Any reason for this?

File: index.ts [hunk 3]
```diff
@@ -229,4 +231,7 @@ async function main() {
   }
 }
 
-main()
+// If module is run as main (ie. from command line)
+if (typeof module !== 'undefined' && !module.parent) {
+  main()
```
Comment (jjperezaguinaga) [inline index.ts:236]: Clever, like it.

File: package.json [hunk 4]
```diff
@@ -11,7 +11,7 @@
     "clean:zip": "rimraf *.zip",
     "clean:all": "yarn clean:lib && yarn clean:hopr-chat && yarn util:clean:all && yarn clean:zip",
     "compile": "yarn clean:hopr-chat && ncc build index.ts -o hopr-chat && copyfiles package.json .env.example start-hopr-chat.bat start-hopr-chat.sh start-hopr-chat.command hopr-chat && yarn util:compile",
-    "test": "jest || echo '⚠️  Currently no tests available'",
```
Comment (jjperezaguinaga) [inline package.json:14]: 👀  don't git blame this one.

Comment (peterbraden) [reply package.json:14]: All good...

**Pull request merged**